mesa/visualization/ModelCreator.py

```diff
@@ -0,0 +1,92 @@
+import solara
+
+from mesa.visualization.solara_viz import split_model_params
+from mesa.visualization.UserParam import Slider
+
+
+@solara.component
+def ModelCreator(model, model_cls, model_params):
```

Comment:
So you can input a model instance, model class and params? I get either the first or the second two are required?

If you input an instance, will it be updated in place, so you can access it from your interpreter later?

Reply:
Actually it expects a reactive version of a model, so that the model from the outside still receives the recreated versions. 

I now removed the model_cls, as it can be resolved from the model instance. 

So now it receives an actual model (as a reactive variable) and user params. So I think ModelCreator isn't a good name